cmd/gateway-gcs.go

```diff
@@ -827,7 +827,7 @@ func (l *gcsGateway) ListMultipartUploads(bucket string, prefix string, keyMarke
 
 // CopyObjectPart - copy part of object to other bucket and object
 func (l *gcsGateway) CopyObjectPart(srcBucket string, srcObject string, destBucket string, destObject string, uploadID string, partID int, startOffset int64, length int64) (info PartInfo, err error) {
-	return PartInfo{}, traceError(NotSupported{})
+	return PartInfo{}, traceError(NotImplemented{})
```

Comment:
Is this intended here in this PR?

Reply:
Will remove..but we need to fix that. Otherwise tests will fail.